app/views/govuk_publishing_components/components/docs/layout_header.yml

```diff
@@ -36,6 +36,29 @@ examples:
       - text: Hidden on desktop
         href: "item-3"
         show_only_in_collapsed_menu: true
+
+  with_left_search_and_navigation:
+    data:
```

Comment:
Can I suggest a description be added here, to explain where this variation on the header will be used?